pkg/utils/kube/kube.go

```diff
@@ -129,10 +129,21 @@ func MustToUnstructured(obj interface{}) *unstructured.Unstructured {
 
 // GetAppInstanceLabel returns the application instance name from labels
 func GetAppInstanceLabel(un *unstructured.Unstructured, key string) string {
```

Comment:
@darshanime , looks like `GetAppInstanceLabel` is not used in GitOps engine. Instead of making changes here please move `GetAppInstanceIdentifier` with dependencies to https://github.com/argoproj/argo-cd/pull/4123 and then we can remove  `GetAppInstanceLabel`  from GitOps engine.